gravitee-apim-console-webui/src/management/api/notifications/notifications-add-notification-dialog/notifications-add-notification-dialog.component.html

```diff
@@ -1,17 +1,42 @@
 <!--
 
     Copyright (C) 2015 The Gravitee team (http://gravitee.io)
-    
+
     Licensed under the Apache License, Version 2.0 (the "License");
     you may not use this file except in compliance with the License.
     You may obtain a copy of the License at
-    
+
             http://www.apache.org/licenses/LICENSE-2.0
-    
+
     Unless required by applicable law or agreed to in writing, software
     distributed under the License is distributed on an "AS IS" BASIS,
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
 
 -->
+
+<h2 mat-dialog-title class="title">New notification</h2>
+
+<form autocomplete="off" [formGroup]="notificationForm" (ngSubmit)="onSubmit()">
+  <mat-dialog-content>
+    <div class="form">
+      <mat-form-field>
+        <mat-label>Name</mat-label>
+        <input matInput formControlName="name" required />
+        <mat-error *ngIf="notificationForm.get('name').hasError('required')">Name is required.</mat-error>
+      </mat-form-field>
+      <mat-form-field>
+        <mat-label>Notifier</mat-label>
+        <mat-select formControlName="notifier">
+          <mat-option *ngFor="let notifier of notifierList" [value]="notifier.id">{{ notifier.name }}</mat-option>
+        </mat-select>
+      </mat-form-field>
+    </div>
+  </mat-dialog-content>
+
+  <mat-dialog-actions class="actions">
+    <button mat-button [mat-dialog-close]="false">Cancel</button>
```

Comment:
```suggestion
    <button mat-button [mat-dialog-close]="">Cancel</button>
```
to return undefined instead of false.

Reply:
Done